Header header
sensor_msgs/Image dispMap
sensor_msgs/PointCloud2 pointCloud
